README.md

```diff
@@ -168,7 +168,7 @@ This is a schematic logging configuration to show log messages from input_nameA
   - `target`: Target host (fqdn). **Required**.
   - `port`: Port; default to 514.
   - `pki_authmode`: Specifying the default network driver authentication mode. Default to `x509/name`.
-  - `permitted_peers`: Hostname, IP addresses or wildcard DNS domain which will be allowed by the `logging` server to connect and send logs over TLS. Default to '\*.{{ logging_domain }}'
+  - `permitted_server`: Hostname, IP addresses, fingerprint(sha1) or wildcard DNS domain which will be allowed to connect and send logs over TLS. Default to '\*.{{ logging_domain }}'
```

Comment:
I suggest something like this: "Hostname, IP address, fingerprint(sha1) or wildcard DNS domain of the server which this client will be allowed to connect and send logs over TLS. Default to '\*.{{ logging_domain }}'"